src/routes/recipes_routes.js

```diff
@@ -1,9 +1,13 @@
 const app = require("express").Router();
 const recipesController = require("../controllers/recipes_controller");
+const {env} = require("../../config");
 
-recipesController.setUp();
+if (env !== "testing") {
+    recipesController.setUp();
```

Comment:
you added an argument "callback" to setup... you need to pass it here if it's indeed required?

Reply:
the callback is for the done callback sent while testing. It is not required here